### DISCUSSION TITLE
Beautiful Tables in Typst – Quarto

### DISCUSSION THREAD

**Discussion by giscus** [quarto-dev/quarto-cli#11421] [Blog]
# Beautiful Tables in Typst – Quarto

You can use HTML Tables with CSS stying in Typst via Quarto.

https://quarto.org/docs/blog/posts/2024-07-02-beautiful-tables-in-typst/

**Comment by CGMossa** [comment]
There is something not right with "5 Solar Zenith". If I click the typst link below it, I get a PDF, with a table that looks better. I don't recognize the code within it.

**Reply by cderv** [reply]
> with a table that looks better

What do you mean by better ? 

>  I don't recognize the code within it.

The code in the python cell ? It seems correct to me and if I try in a .qmd document I get the same table 🤔

**Reply by CGMossa** [reply]
![Screenshot_20240704-120801.png](https://github.com/quarto-dev/quarto-web/assets/1063624/4b2cd816-26c8-499f-a5f3-e9c06f93b053)

![Screenshot_20240704-120747.png](https://github.com/quarto-dev/quarto-web/assets/1063624/39c93447-d8d5-420a-8b1d-d8213c3a12c1)

I'm seeing two different tables.
The inset should be increased a bit, but it is somehow ignored in one, but not the other.

**Reply by cderv** [reply]
Oh I see, they slightly differ on style. This is probably due to screenshot of the PDF table that we include as preview in blog post. 

Though the overall table is good, with the CSS color applying in Typst. This is small difference in preview in article versus the real rendered document. We'll see if we can fix. 

It could be a limitation of the automated screenshot process cc @gordonwoodhull

**Reply by CGMossa** [reply]
Thanks for clarifying, and also making sense of my feedback. I appreciate that a lot.
Good luck.

**Reply by gordonwoodhull** [reply]
Odd, I’ll have to do a thorough check to see if there are other differences between the snapshots and the documents. 

In this case, it has chosen a completely different font.

Currently we do not try to translate padding to inset. It’s possible, but the quality was not good enough to release.

I am aware that the font often feels too big in Typst. This is despite using the standard conversion of points = 0.75 pixels. We might want to expose this ratio as a parameter.

Thanks for taking a close look at these results!
